mcs/class/System/Test/System.Net.Security/SslStreamTest.cs

```diff
@@ -93,6 +171,167 @@ void AuthenticateClientAndServer (bool server, bool client)
 		}
 	}
 
+	[Category("NotWorking")]    // Works in .Net, doesn't work in mono.
+	[Test()]
+	public void SelfCompatibilityTest()
+	{
+		#if __MonoCS__
```

Comment:
Hey,

Mono.Security.dll works fine on Win32, even when executed on MS.NET, so it
can be used to create unit tests (without requiring any special defines).

Sebastien

On Tue, Jan 6, 2015 at 6:37 AM, Edward Ned Harvey <notifications@github.com>
wrote:

> In mcs/class/System/Test/System.Net.Security/SslStreamTest.cs
> <https://github.com/mono/mono/pull/1490#discussion-diff-22519301>:
>
> > @@ -93,6 +171,167 @@ void AuthenticateClientAndServer (bool server, bool client)
> >  		}
> >  	}
> >
> > +	[Category("NotWorking")]    // Works in .Net, doesn't work in mono.
> > +	[Test()]
> > +	public void SelfCompatibilityTest()
> > +	{
> > +		#if __MonoCS__
>
> I may have posted this in the wrong place before - here it is again - The
> issue that requires the #if MonoCS is not detection of execution platform -
> the code needs to compile without Mono.Security reference, when compiled
> for .Net. This was discussed on the mono-dev list
> http://lists.ximian.com/pipermail/mono-devel-list/2014-December/042533.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mono/mono/pull/1490/files#r22519301>.
>